include/kf/UString.h

```diff
@@ -75,8 +75,8 @@ namespace kf
 
             if (newByteLength > 0)
             {
-#pragma warning(suppress : 4996) // ExAllocatePoolWithTag is deprecated, use ExAllocatePool2
 #pragma warning(suppress: 28160) // Must succeed pool allocations are forbidden. Allocation failures cause a system crash.
+#pragma warning(suppress: 4996) // ExAllocatePoolWithTag is deprecated, use ExAllocatePool2
```

Comment:
Why do we need this change?

Reply:
The warning is not suppressed, fixed in another way